tests/clients/test_report.py

```diff
@@ -0,0 +1,213 @@
+# Copyright 2018 ASI Data Science
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+import datetime
+
+import pytest
+from marshmallow import ValidationError, EXCLUDE
+
+from sherlockml.clients.report import (
+    Report,
+    ReportSchema,
+    ReportWithVersions,
+    ReportWithVersionsSchema,
+    ReportVersion,
+    ReportClient,
+)
+
+from tests.clients.fixtures import PROFILE
+
+
+PROJECT_ID = uuid.uuid4()
+USER_ID = uuid.uuid4()
+REPORT_ID = uuid.uuid4()
+VERSION_ID = uuid.uuid4()
+PROJECT_ID = uuid.uuid4()
+
+ACTIVE_VERSION = ReportVersion(
+    created_at=datetime.datetime(2018, 10, 3, 9, 23, 5, 0),
```

Comment:
Can we test this with datetimes with a timezone? We normally pass around zoned datetimes for explicitness, and it would be good for these tests to reflect that.

See for an example `tests/clients/test_workspace.py` line 39.

Reply:
I'm just trying this (it's making the tests fail because deserialised responses don't have a TZ).

So should the deserialisation assign UTC as the timezone by default so that the python object has a datetime with timezone?

Reply:
We expect the API to return a datetime with a timezone (a trailing Z indictating UTC as in the tests you've implemented), which marshmallow correctly interprets.